rpcs3/Emu/Cell/Modules/cellSysutil.cpp

```diff
@@ -497,14 +528,26 @@ error_code cellSysutilUnregisterCallback(u32 slot)
 	return CELL_OK;
 }
 
-bool g_bgm_playback_enabled = true;
+struct bgm_manager
+{
+	std::mutex mtx;
```

Comment:
Use shared_mutex for consistency. Std one was known to be noticeably slow.

Reply:
done